switchapi/src/switch_pd.c

```diff
@@ -7827,7 +7827,11 @@ switch_pd_mirror_session_update(switch_device_t device,
     switch_api_mirror_info_t *api_mirror_info = NULL;
 
     api_mirror_info = &mirror_info->api_mirror_info;
+#ifndef BMV2
+    status = p4_pd_dc_mirror_session_update(
```

Comment:
I think Mahesh is going to make changes to Tofino mirroring apis to remove _dc_ from the name. This change will not be needed once he is done.